spec/text-editor-component-spec.js

```diff
@@ -1651,6 +1651,106 @@ describe('TextEditorComponent', function () {
     })
   })
 
+  describe('block decorations rendering', function () {
```

Comment:
since this will be transpiled by babel, why not preferring the `() => {}` over `function() {}`